Update expo-cli docs (#3752)
expo/expo: docs/pages/versions/unversioned/workflow/expo-cli.md

title: Expo CLI
---

> Expo CLI is a command line interface that enhances your workflow by empowering developers to perform actions such as:

Reviewer: ```suggestion
Expo CLI is a command line app that is the main interface between a developer and Expo tools. You'll use it for a variety of tasks, such as:
```

> - Creating a new Expo project

Reviewer: ```suggestion
- Creating new projects
```

> - Serving your app so you can begin development

Reviewer: ```suggestion
- Developing your app: running the project server, viewing logs, opening your app in a simulator 
```

> - Publishing your JS and assets

Reviewer: ```suggestion
- Publishing your app JavaScript and other assets and managing releasing them over the air
```

> - Building `apk` and `ipa` files to be uploaded to the App Store and Play Store

Reviewer: ```suggestion
- Building binaries (`apk` and `ipa` files) to be uploaded to the App Store and Play Store
```

> - Uploading Apple Credentials and Google Keystores

Reviewer: ```suggestion
- Managing Apple Credentials and Google Keystores
```

> It also includes a web-based interface, named Expo Dev Tools, which enables you to use some of the most often used features from a quick-to-use graphical interface. We’ve only scratched the surface of what expo-cli can do so far. Be sure to check out all the possible commands below!

Reviewer: You may use the CLI in your terminal or use the web based interface (it opens automatically by default, or you can press `d` from the CLI to open it on demand). The web interface enables you to use some of the most often used features from a quick-to-use graphical interface. We’ve only scratched the surface of what expo-cli can do so far. Be sure to check out all the possible commands below!
